etc/afpd/status.c

```diff
@@ -520,12 +528,6 @@ void status_init(AFPObj *dsi_obj, AFPObj* asp_obj, DSI *dsi)
         icon = sunlogo_icon;
         iconlen = sizeof(sunlogo_icon);
     }
-#ifndef NO_DDP
-    else if (asp) {
-        icon = apple_atalk_icon;
-        iconlen = sizeof(apple_atalk_icon);
-    }
```

Comment:
I want to turn this into a manual option instead, add it to the selection of icons that you can choose. Optional for this PR, though. I can do it later.